lang/c++/test/SchemaTests.cc

```diff
@@ -154,6 +205,19 @@ static void testCompile(const char* schema)
     compileJsonSchemaFromString(std::string(schema));
 }
 
+// Test that the JSON output from a valid schema matches the JSON that was 
+// used to construct it, apart from whitespace changes.
+static void testRoundTrip(const char* schema)
+{
+    BOOST_CHECKPOINT(schema);
```

Comment:
Other tests have been updated to use BOOST_TEST_CHECKPOINT, please update this as well.